src/pages/documentation/mobile-core/api-reference.md

```diff
@@ -0,0 +1,545 @@
+
+Xamarin
+
+<GetApplicationXamarin/>
+
+## getLogLevel
+
+This API gets the current log level being used in the SDK.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="6"/>
+
+Android
+
+<GetLogLevelAndroid/>
+
+iOS (AEP 3.x)
+
+<GetLogLevelIosAEP/>
+
+iOS (ACP 2.x)
+
+<GetLogLevelIosACP/>
+
+React Native
+
+<GetLogLevelReactNative/>
+
+Unity
+
+<GetLogLevelUnity/>
+
+Xamarin
+
+<GetLogLevelXamarin/>
+
+## getSdkIdentities
+
+The following SDK identities are locally stored:
+
+* Company Context - IMS Org IDs
+* Experience Cloud ID (MID)
+* User IDs
+* Integration codes (ADID, push IDs)
+* Data source IDs (DPID, DPUUID)
+* Analytics IDs (AVID, AID, VID, and associated RSIDs)
+* Target legacy IDs (TNTID, TNT3rdpartyID)
+* Audience Manager ID (UUID)
+
+To retrieve data as a JSON string from the SDKs and send this data to your servers, use the `getSdkIdentities` method.
+
+<InlineAlert variant="warning" slots="text"/>
+
+You must call the API below and retrieve identities stored in the SDK, **before** the user opts out.<br/><br/>This API does **not** include the identities stored in the Edge Identity extension. To retrieve the identities from the Edge Identity extension, use [getIdentities](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/09dd71f04d377c356dd24aac9b89ed0fffc1cf63/using-mobile-extensions/adobe-edge-identity/adobe-edge-identity-api-reference.md#getidentities). (FIX LINK)
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="3"/>
+
+Android
+
+<GetSdkIdentitiesAndroid/>
+
+iOS (AEP 3.x)
+
+<GetSdkIdentitiesIosAEP/>
+
+iOS (ACP 2.x)
+
+<GetSdkIdentitiesIosACP/>
+
+## log
+
+This is the API used to log from the SDK.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="5"/>
+
+Android
+
+<LogAndroid/>
+
+iOS (AEP 3.x)
+
+<LogIosAEP/>
+
+iOS (ACP 2.x)
+
+<LogIosACP/>
+
+React Native
+
+<LogReactNative/>
+
+Xamarin
+
+<LogXamarin/>
+
+## registerExtension(s)
+
+Extensions are registered with Mobile Core so that they can dispatch and listen for events.
+
+<InlineAlert variant="warning" slots="text"/>
+
+Extension registration is **mandatory**. Attempting to make extension-specific API calls without registering the extension will lead to undefined behavior.
+
+The following code snippets demonstrate how you can import and register the Mobile Core and Profile extensions. You can also see, for reference, how Identity, Lifecycle, Signal, Profile, and other extensions are imported and registered.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="8"/>
+
+Android
+
+<RegisterExtensionAndroid/>
+
+iOS (AEP 3.x)
+
+<RegisterExtensionIosAEP/>
+
+iOS (ACP 2.x)
+
+<RegisterExtensionIosACP/>
+
+React Native
+
+<RegisterExtensionReactNative/>
+
+Flutter
+
+<RegisterExtensionFlutter/>
+
+Cordova
+
+<RegisterExtensionCordova/>
+
+Unity
+
+<RegisterExtensionUnity/>
+
+Xamarin
+
+<RegisterExtensionXamarin/>
+
+## registerURLHandler 
+
+FIX LINK
```

Comment:
This should be removed.

Reply:
The fix links are there for me to keep track of links I need to fix going forward, since it's a link to a page I haven't migrated over yet (so it's there by design at the moment). Since we're not making this public yet, I have that there on purpose so I can know what links to update once I have migrated the page over.